### PR TITLE
Update fr.yml for 1.0.11 Obinslab starter version

### PR DESCRIPTION
**Pull request by ochbob**
minor fix + updated fr translation for 1.0.11 Obinslab starter version

### CODE REVIEW

**Comment by ochbob** [comment]
Sorry for the two previous closed pull request i'm a bit new with git.

**Comment by obinslab** [comment]
Great work. Could you please format the code with [prettier-vscode](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)?

**Comment by ochbob** [comment]
Should be done, let me know if it's good for you.

**Comment by obinslab** [comment]
Perfect, thank you!

File: locales/fr.yml [hunk 1]
```diff
@@ -91,11 +91,12 @@ preferences.help.user-manual: Manuel utilisateur
 preferences.about: À propos
 
 # Écran de démarrage
-home.welcome: Bonjour le monde
+home.welcome: Bienvenue
 home.please_connect_kbd_with_cable: Veuillez connecter votre appareil via USB
-home.open_device_failed: Échec d'ouverture du périphérique
+home.open_device_failed: Échec de connexion au périphérique
 home.open_device_failed_linux:
```
Comment (obinslab) [inline locales/fr.yml]: Here, the key and value is not in one line

Comment (ochbob) [reply locales/fr.yml]: Thank, you I change the faulty lines, hope that is what you want :)

**Pull request merged**